aw/ViewControllers/Reaches/Detail/ReachDetailMapViewController.swift

```diff
@@ -34,6 +34,57 @@ extension ReachDetailMapViewController {
     func initialize() {
         mapView.mapType = .hybrid
         mapView.delegate = self
+
+        let keyView = UIView()
```

Comment:
Minor: the code would be more readable if you pull this out into a view class, or at least into another function. You have also built something similar for the other map view, not sure if you can reuse code.

Reply:
Good call, it's reusable!